boards/arm/sam4s_xplained/ext_ram.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) Justin Watson 2017
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <init.h>
+#include <nanokernel.h>
+#include <pinmux.h>
+#include <soc.h>
+#include <sys_io.h>
+
+/* Static Memory Controller (SMC) */
```

Comment:
Yeah, I can take it out of the patch. I was thinking that after I responded to you earlier. There is no precedent currently for external RAM. I did this for a project I'm working on. I can just keep it in my personal branch.